hugo/assets/scss/components/teaser.scss

```diff
@@ -29,45 +27,129 @@
 
         color: $c-blue--dark;
         font-size: 1.5rem;
-        margin: 0 0 0.4rem;
+        margin: 0 0 .4rem;
     }
 
     &__meta {
         color: var(--teaser-meta-color, $c-grey--dark);
-        font-size: 0.8125rem;
+        font-size: .8125rem;
```

Comment:
please use 0. notation 
we can't force this in editrconfig but prettier etc should do this automatically